l10n_it_reverse_charge/tests/test_rc.py

```diff
@@ -408,3 +408,29 @@ def test_intra_EU_cancel_and_draft(self):
         self.assertEqual(invoice.state, 'cancel')
         invoice.action_invoice_draft()
         self.assertEqual(invoice.state, 'draft')
+
+    def test_new_refund_flag(self):
+        """Check that the lines of a new refund have the RC flag properly set.
+        """
+        supplier_extraEU = self.partner_model.create({
```

Comment:
`supplier_extraEU` could be moved to `setUp`

Reply:
@eLBati grazie della review, fatto in https://github.com/OCA/l10n-italy/pull/1155/commits/b8dbcca91fb89e842a4799bcb8d5f6d94d3ef5cf